Random failures in area_test.go

I noticed a few PR checks failed with unrelated test failures in area_test.go:
```
FAIL: TestListChildrenOfParents (0.01s)
	assertions.go:241: 
                        
	Error Trace:	area_test.go:187
		
	Error:		Expected nil, but got: errors.BadParameterError{parameter:"Name", value:"Space 1", expectedValue:"unique", hasExpectedValue:true}

	--- FAIL: TestListParentTree (0.02s)
	assertions.go:241: 
                        
	Error Trace:	area_test.go:326
		
	Error:		Expected nil, but got: errors.BadParameterError{parameter:"Name", value:"Space 1", expectedValue:"unique", hasExpectedValue:true}
		

	assertions.go:241: 
                        
	Error Trace:	area_test.go:333
		
	Error:		Expected nil, but got: &pq.Error{Severity:"ERROR", Code:"23503", Message:"insert or update on table \"areas\" violates foreign key constraint \"areas_space_id_spaces_id_fk\"", Detail:"Key (space_id)=(06d125c4-4e23-4352-a100-61b32a31e7a6) is not present in table \"spaces\".", Hint:"", Position:"", InternalPosition:"", InternalQuery:"", Where:"", Schema:"public", Table:"areas", Column:"", DataTypeName:"", Constraint:"areas_space_id_spaces_id_fk", File:"ri_triggers.c", Line:"3326", Routine:"ri_ReportViolation"}
```

@sbose78 ?

--- a/area/area_test.go
+++ b/area/area_test.go
@@ -47,11 +47,11 @@ func (test *TestAreaRepository) TestCreateArea() {
 
 	name := "Area 21"
 	newSpace := space.Space{
-		Name: "Space 1",
+		Name: uuid.NewV4().String(),
 	}
 	repoSpace := space.NewRepository(test.DB)
 	space, err := repoSpace.Create(context.Background(), &newSpace)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 
 	i := area.Area{
 		Name:    name,
@@ -78,11 +78,11 @@ func (test *TestAreaRepository) TestCreateChildArea() {
 	repo := area.NewAreaRepository(test.DB)
 
 	newSpace := space.Space{
-		Name: "Space 1",
+		Name: uuid.NewV4().String(),
 	}
 	repoSpace := space.NewRepository(test.DB)
 	space, err := repoSpace.Create(context.Background(), &newSpace)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 
 	name := "Area #24"
 	name2 := "Area #24.1"
@@ -120,11 +120,11 @@ func (test *TestAreaRepository) TestListAreaBySpace() {
 	repo := area.NewAreaRepository(test.DB)
 
 	newSpace := space.Space{
-		Name: "Space #1",
+		Name: uuid.NewV4().String(),
 	}
 	repoSpace := space.NewRepository(test.DB)
 	space1, err := repoSpace.Create(context.Background(), &newSpace)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 
 	var createdAreaIds []uuid.UUID
 	for i := 0; i < 3; i++ {
@@ -141,9 +141,11 @@ func (test *TestAreaRepository) TestListAreaBySpace() {
 	}
 
 	newSpace2 := space.Space{
-		Name: "Space #2",
+		Name: uuid.NewV4().String(),
 	}
 	space2, err := repoSpace.Create(context.Background(), &newSpace2)
+	require.Nil(t, err)
+
 	err = repo.Create(context.Background(), &area.Area{
 		Name:    "Other Test area #20",
 		SpaceID: space2.ID,
@@ -180,11 +182,11 @@ func (test *TestAreaRepository) TestListChildrenOfParents() {
 	var createdAreaIDs []uuid.UUID
 
 	newSpace := space.Space{
-		Name: "Space 1",
+		Name: uuid.NewV4().String(),
 	}
 	repoSpace := space.NewRepository(test.DB)
 	space, err := repoSpace.Create(context.Background(), &newSpace)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 
 	// *** Create Parent Area ***
 	i := area.Area{
@@ -254,11 +256,11 @@ func (test *TestAreaRepository) TestListImmediateChildrenOfGrandParents() {
 	name3 := "Area #240.1.3"
 
 	newSpace := space.Space{
-		Name: "Space 1",
+		Name: uuid.NewV4().String(),
 	}
 	repoSpace := space.NewRepository(test.DB)
 	space, err := repoSpace.Create(context.Background(), &newSpace)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 
 	// *** Create Parent Area ***
 
@@ -319,11 +321,11 @@ func (test *TestAreaRepository) TestListParentTree() {
 	name2 := "Area #240.1"
 
 	newSpace := space.Space{
-		Name: "Space 1",
+		Name: uuid.NewV4().String(),
 	}
 	repoSpace := space.NewRepository(test.DB)
 	space, err := repoSpace.Create(context.Background(), &newSpace)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 	// *** Create Parent Area ***
 	i := area.Area{
 		Name:    name,

